Mapping/AutoRouteDefinition.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\RoutingAuto\Mapping;
+
+/**
+ * Represents the definition of an auto route.
+ */
+class AutoRouteDefinition
+{
+    /**
+     * This constant will be used as the auto route definition's
+     * "name" when the deprecated string value for the "uri-schema" is used.
+     */
+    const DEPRECATED_NAME = '_default_';
```

Comment:
I think we should mark this as deprecated.

Also, I would prefer a name like `DEFAULT_NAME`.